cli/tools/repl/channel.rs

```diff
@@ -34,14 +35,8 @@ pub fn rustyline_channel(
 }
 
 pub enum RustylineSyncMessage {
-  PostMessage {
-    method: String,
-    params: Option<Value>,
-  },
-  LspCompletions {
-    line_text: String,
-    position: usize,
-  },
+  PostMessage { method: String, params: Value },
+  LspCompletions { line_text: String, position: usize },
```

Comment:
Seems like unrelated change

Reply:
nope, since i the `Option<>` from `params` it reformatted it all